.travis.yml

```diff
@@ -1,24 +1,51 @@
 sudo: required
 language: go
 go:
-      - "1.12"
+      - "1.12.15"
 before_install:
     - export GOPATH=$HOME/go
     - export PATH=$HOME/usr/local/go/bin:$GOPATH/bin:$PATH
     - mkdir -p $GOPATH/src/github.com/ovn-org
     - mv $TRAVIS_BUILD_DIR $GOPATH/src/github.com/ovn-org/ovn-kubernetes
     - cd $GOPATH/src/github.com/ovn-org/ovn-kubernetes
-install:
-    - eval "$(gimme 1.12)"
-script:
-    - pushd go-controller
-    -    make
-    -    make windows
-    -    make gofmt
-    -    make install.tools
-    -    make lint
-    -    make check
-    - popd
-    - pushd dist/images
-    -    if [ -n "$(git diff --stat origin/master.. | grep dist/images/Dockerfile)" ]; then make all; fi
-    - popd
+
+jobs:
+  include:
+    - stage: build
+      script:
+        - pushd go-controller
+        -    make
+        -    make windows
+        -    make gofmt
+        -    make install.tools
+        -    make lint
+        -    make check
+        - popd
+        - pushd dist/images
+        -    if [ -n "$(git diff --stat origin/master.. | grep dist/images/Dockerfile)" ]; then make all; fi
+        - popd
+    - stage: e2e-kind-ovn
+      before_script:
+        - export GO111MODULE="on"
+        - git clone https://github.com/kubernetes/test-infra.git $GOPATH/src/k8s.io/test-infra
+        - pushd $GOPATH/src/k8s.io/test-infra/kubetest
```

Comment:
Building this from master seems scary... ah, but they have no tagged releases. Blah. We should get them to tag a known-working version...

Reply:
Yeah their readme says to always use head. I wish they just had a binary download release...it takes a long time to build it. I was thinking about not even using it and just calling e2e tests directly, but kubetest has some nice validation for KIND deployments so I thought it was useful. If we can split it up into its own job, upload the binary somewhere, then download it in the e2e job, it will give us more time to work with in the e2e job.

Reply:
In the latest patch, I just download the binary from one i built and uploaded to chunk.io. We can periodically update it in the future.